packages/react-native-reanimated/android/build.gradle

```diff
@@ -445,20 +446,31 @@ def assertMinimalReactNativeVersion = task assertMinimalReactNativeVersionTask {
     }
 }
 
-task prepareHeadersForPrefab(type: Copy) {
-    from("$projectDir/src/main/cpp/reanimated")
+task prepareWorkletsHeadersForPrefabs(type: Copy) {
+    from("$projectDir/src/worklets/cpp")
+    from("$projectDir/../Common/cpp/worklets/NativeModules")
+    from("$projectDir/../Common/cpp/worklets/Registries")
+    from("$projectDir/../Common/cpp/worklets/SharedItems")
+    from("$projectDir/../Common/cpp/worklets/Tools")
+    from("$projectDir/../Common/cpp/worklets/WorkletRuntime")
+    include("*.h")
+    into(workletsPrefabHeadersDir)
+}
+
+task prepareReanimatedHeadersForPrefabs(type: Copy) {
+    // Reanimated
```

Comment:
```suggestion
    // Reanimated
    from("$projectDir/src/main/cpp/reanimated")
```

Reply:
I'd rather make this change in a separate PR.

Reply:
https://github.com/software-mansion/react-native-reanimated/pull/6566